### ISSUE TITLE
Bug with apostrophe in the query filter

### ISSUE THREAD

**Issue by Goodmain** [RonasIT/laravel-helpers#132]
We have a bug when the apostrophe symbol appears in the `query` filter.

```sh
PDOException: SQLSTATE[42601]: Syntax error: 7 ERROR:  syntax error at or near "s"
LINE 1: ... "is_hidden" in ($1) and (("name"::text ilike '%wooden's%'))
                                                                  ^ in /app/vendor/laravel/framework/src/Illuminate/Database/Connection.php:419
Stack trace:
...
#11 /app/vendor/ronasit/laravel-helpers/src/Traits/SearchTrait.php(49): Illuminate\Database\Eloquent\Builder->paginate(10, Array, 'page', 1)
#12 /app/vendor/ronasit/laravel-helpers/src/Traits/SearchTrait.php(165): RonasIT\Support\Repositories\BaseRepository->paginate()
#13 [internal function]: RonasIT\Support\Repositories\BaseRepository->getSearchResults()
#14 /app/vendor/ronasit/laravel-helpers/src/Services/EntityService.php(25): call_user_func_array(Array, Array)
#15 /app/app/Services/PropertyGroupService.php(37): RonasIT\Support\Services\EntityService->__call('getSearchResult...', Array)
#16 /app/app/Http/Controllers/PropertyGroupController.php(41): App\Services\PropertyGroupService->search(Array)
#17 /app/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): App\Http\Controllers\PropertyGroupController->search(Object(App\Http\Requests\PropertyGroups\SearchPropertyGroupsRequest), Object(App\Services\PropertyGroupService))

```

**Comment by ulin-evgeny** [comment]
Done.

**Comment by DenTray** [comment]
Looks good, released in 3.0.0-beta
